src/App.js

```diff
@@ -11,8 +11,8 @@ export default function App() {
 			<BrowserRouter>
 				<Routes>
 					<Route path="/">
-						<Route index element={<Home />} />
-						<Route path="about/" element={<About />} />
+						<Route index element={<HomePage />} />
+						<Route path="calendar/" element={<CalendarPage />} />
```

Comment:
I don't think you need the slash at the end of the path, do you? :)
```suggestion
						<Route path="calendar" element={<CalendarPage />} />
```